Darktrace/darktrace-threat-visualizer/ingest/parser.yml

```diff
@@ -52,17 +54,15 @@ stages:
           observer.name: "Darktrace"
           observer.product: "Threat visualizer"
           host.ip: >
-            {% if json_event.message.device.ips is not in [[], Null]%}
```

Comment:
Changed this, because `host.ip: []` confuses smart descriptions generator and we get smart desc with an empty block for `host.ip` in result

Though, it's hard to see because of broken smart descriptions in PR 😅